File: Unidad3/Java/U3p13.java
package com.mycompany.u3p13;
import java.util.Scanner;
public class U3p13 {

    public static void main(String[] args) {
        Scanner nose = new Scanner (System.in);
        int c, total, num, negativos = 0, positivos = 0;
        System.out.println("Ingresa el total de numeros a procesar: ");
        total=nose.nextInt ();
        
        for (c=1; c<=total; c++){
        System.out.println("Ingresa tu numero: ");
        num=nose.nextInt();
        if (num>0){
            positivos++;
        }
        else if(num<0){
          negativos++;
        }
                System.out.println("Cantidad de numeros Positivos="+positivos);
                System.out.println("Cantidad de numeros Negativos="+negativos);
    }
}
}

File: Unidad3/C#/u3p13.cs
using System;
class Program
{
    static void Main()
    {
        int total, num, negativos = 0, positivos = 0;
        Console.WriteLine("Ingresa el total de numeros a procesar: ");
        total = Convert.ToInt32(Console.ReadLine());

        for (int c = 1; c <= total; c++)
        {
            Console.WriteLine("Ingresa tu numero: ");
            num = Convert.ToInt32(Console.ReadLine());
            if (num > 0)
            {
                positivos++;
            }
            else if (num < 0)
            {
                negativos++;
            }
            Console.WriteLine("Cantidad de numeros Positivos=" + positivos);
            Console.WriteLine("Cantidad de numeros Negativos=" + negativos);
        }
    }
}
